docs/reference/synonyms/apis/get-synonyms-set.asciidoc

```diff
@@ -0,0 +1,78 @@
+[[get-synoyms-set]]
+=== Get Synonyms Set
+
+beta::[]
+
+++++
+<titleabbrev>Get Synonyms Set</titleabbrev>
+++++
+
+Retrieves a Synonyms Set.
+
+[[get-synonyms-set-request]]
+==== {api-request-title}
+
+`GET _synonyms/<synonyms_set>`
+
+[[get-synonyms-set-prereqs]]
+==== {api-prereq-title}
+
+Requires the `manage_search_synonyms` cluster privilege.
+
+[[get-synonyms-set-path-params]]
+==== {api-path-parms-title}
+
+`<synonyms_set>`::
+(Required, string)
+Synonyms set identifier to retrieve.
+
+[[get-synonyms-set-query-params]]
+==== {api-query-parms-title}
+
+`from`::
+(Optional, integer) Starting offset for synonyms rules to retrieve. Defaults to `0`.
+
+`size`::
+(Optional, integer) Specifies the maximum number of synonyms rules to retrieve. Defaults to `10`.
+
+[[get-synonyms-set-response-codes]]
+==== {api-response-codes-title}
+
+`400`::
+The `synonyms_set` identifier was not provided.
+
+`404` (Missing resources)::
+No Synonyms Set with the identifier `synonyms_set` was found.
+
+[[get-synonyms-set-example]]
+==== {api-examples-title}
+
+The following example retrieves a Synonyms Set called `my-synonyms-set`:
+
+[source,console]
+----
+GET _synonyms/my-synonyms-set
+----
+// TEST[skip:TBD]
+
+The Synonyms Set information returned will include the total number of synonyms rules that the synonyms set contains, and the synonyms rules according to the `from` and `size` parameters.
+
+A sample response:
+
+[source,console-result]
+----
+{
+  "count": 2,
+  "synonyms_set": [
+    {
+      "id": "test-1",
+      "synonyms": "hello, hi"
+    },
+    {
+      "id": "test-2",
+      "synonyms": "test => check"
+    }
+  ]
+}
+----
+// TEST[skip:TBD]
```

Comment:
What is the reason we are skipping running these doc tests?

Reply:
I hadn't started working on tests yet - I'll give it a go